src/components/profile/UserDetailForm/index.jsx

```diff
@@ -1,5 +1,117 @@
 import React from "react";
+import { Col } from "react-bootstrap";
+import countries from "./countries.json";
+import Input from "../../form/Input";
+import "./index.scss";
 
-export default function UserDetailForm() {
-  return <div>UserDetail</div>;
+export default function UserDetail({ register }) {
+  return (
+    <Col xs={12} className="userDetailSection">
+      <div className="formInputGroup">
+        <Input
+          type="text"
+          placeholder="Your first name"
+          name="firstName"
+          ref={register()}
+          label="Your first Name"
+        />
+
+        <Input
+          type="text"
+          placeholder="Your last name"
+          name="lastName"
+          ref={register()}
+          label="Last Name"
+        />
+      </div>
+
+      <div className="formInputGroup">
+        <Input
+          type="text"
+          placeholder="Your nationality"
+          name="nationality"
+          label="Your nationality"
+          ref={register()}
+        />
+
+        <Input
+          type="tel"
+          placeholder="+90 552 477 25 15"
+          name="phoneNumber"
+          label="Phone Number"
+          ref={register()}
+        />
+      </div>
+
+      <div className="formInputGroup">
+        <div className="formInput">
+          <label htmlFor="sex">
+            Sex
+            <select name="sex" id="sex" ref={register()}>
+              <option value="female">Female</option>
+              <option value="male">Male</option>
+            </select>
+          </label>
+        </div>
+
+        <Input
+          type="date"
+          name="birthdate"
+          id="birthdate"
+          label="Birthdate"
+          ref={register()}
+        />
+      </div>
+
+      <Input
+        type="text"
+        placeholder="1234 UTK  str.  Dumplinar Mah."
+        name="address"
+        label="Address"
+        ref={register()}
+      />
+
+      <div className="formInputGroup">
+        <Input
+          type="text"
+          placeholder="Bursa"
```

Comment:
if you are allowing people to pick their country and expect them to be intl they probably don't know bursa. use London or something if you want to do this

Reply:
As we are targeting Turkey, so I think Bursa is known at this point. I can work on that an use some library to link this with the country but it will take much time I do not have :(( If it's up to me, I prefer to like all the location parts together

Reply:
I will change it to Istanbul